test/libsolidity/semanticTests/inlineAssembly/keccak256_optimizer_bug_different_memory_location.sol

```diff
@@ -0,0 +1,16 @@
+contract C {
+    function f() public view returns (bool ret) {
+        assembly {
+            let x := calldataload(0)
+            mstore(0, x)
+            mstore(0x20, x)
+            let a := keccak256(0, 4)
+            let b := keccak256(0x20, 8)
```

Comment:
Do you think there is value in adding an example where keccak256 arguments are compile-time constants but not literals?

Reply:
You mean an example or a test? I don't think it's too useful.